### PR TITLE
5881 total rep stake

### PR DESCRIPTION
**Pull request by bthaile**
small PR to use augur-node's convenient method, rest of PR on other CH task

ended up being all there is needed for 5838 as well.
https://app.clubhouse.io/augur/story/5838/dispute-view-header

### CODE REVIEW

File: src/modules/reporting/actions/load-reporting-window-bounds.js [hunk 1]
```diff
@@ -3,33 +3,19 @@ import { updateReportingWindowStats } from 'modules/reporting/actions/update-rep
...
-    })
-
-    augur.api.FeeWindow.getEndTime({ tx: { to: reportingWindow } }, (err, endTime) => {
+  augur.augurNode.submitRequest(
+    'getFeeWindowCurrent',
+    {
+      universe: universe.id,
+      reporter: loginAccount.address
+    }, (err, result) => {
       if (err) return
```
Comment (tinybike) [inline src/modules/reporting/actions/load-reporting-window-bounds.js]: Hm, should there be a callback here (or at least a `console.error`), so if there is an error it propagates to the caller?  Same question for other `if (err) return` lines.

Comment (bthaile) [reply src/modules/reporting/actions/load-reporting-window-bounds.js]: Yes we need to at least log it. If error we wint update fee window properties. I’ll fix

Comment (bthaile) [reply src/modules/reporting/actions/load-reporting-window-bounds.js]: added callback to log error

File: src/env-dev.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
     "ws": "ws://127.0.0.1:8546"
   },
   "network-id": 4,
-  "auto-login": true,
+  "auto-login": false,
```
Comment (tinybike) [inline src/env-dev.json]: Was this changed on purpose?

Comment (bthaile) [reply src/env-dev.json]: nope not meant to be changed.

**Comment by bthaile** [comment]
removed the change to env-dev.json file.

**Pull request merged**